### ISSUE TITLE
"nan" should be a non-dev dependency

### ISSUE THREAD

**Issue by jokester** [gagern/node-libtidy#12]
I'm getting following error when installing v0.3.0 from npm. Installing `nan` beforehand fixed it.

```
% npm install libtidy

> libtidy@0.3.0 install /Users/o_kan/aaa/node_modules/libtidy
> node-gyp rebuild

module.js:327
    throw err;
    ^

Error: Cannot find module 'nan'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at [eval]:1:1
    at Object.exports.runInThisContext (vm.js:54:17)
    at Object.<anonymous> ([eval]-wrapper:6:22)
    at Module._compile (module.js:409:26)
    at node.js:579:27
    at nextTickCallbackWith0Args (node.js:420:9)
gyp: Call to 'node -e "require('nan')"' returned exit status 1 while in binding.gyp. while trying to load binding.gyp
```

**Comment by jokester** [comment]
If moving it from `devDependencies` to `dependencies` is fine, I can do something after work :o .

**Comment by gagern** [comment]
Fixed in v0.3.1. Thanks for the report!
